pkg/ice/agent.go

```diff
@@ -646,22 +647,60 @@ func (a *Agent) handleInboundControlling(m *stun.Message, localCandidate, remote
 
 // handleInbound processes traffic from a remote candidate
 func (a *Agent) handleInbound(m *stun.Message, local *Candidate, remote net.Addr) {
+	iceLog.Tracef("inbound STUN from %s to %s", remote.String(), local.String())
 	remoteCandidate := a.findRemoteCandidate(local.NetworkType, remote)
 	if remoteCandidate == nil {
-		iceLog.Debugf("Could not find remote candidate for %s ", remote)
-		return
-	}
-
-	remoteCandidate.seen(false)
+		//iceLog.Debugf("Could not find remote candidate for %s ", remote)
```

Comment:
Small suggestion: Maybe move this entire section to a separate method so you can return errors to make the code flatter:
```go
if err != nil {
  return err 
}
doSomething()
```
You can still log the error in `handleInbound`.

Reply:
Good idea. Will do.